generators/upgrade/generator.mjs

```diff
@@ -30,10 +30,9 @@ import BaseGenerator from '../base/index.mjs';
 import { upgradeFiles } from '../cleanup.mjs';
 import constants from '../generator-constants.cjs';
 import statistics from '../statistics.cjs';
-import blueprintUtils from '../../utils/blueprint.cjs';
+import { parseBluePrints } from '../../utils/blueprint.mjs';
```

Comment:
`parseBlueprint`?

Reply:
The original is parseBluePrints.

Reply:
Yes, but we're preparing V8 aren't we?

Reply:
Don’t analyzed if we need to rename. Can rename later if makes sense.